tests/foreman/cli/test_organization.py

```diff
@@ -55,26 +55,31 @@ def valid_labels_list():
         gen_string('alpha'),
         gen_string('alphanumeric'),
         gen_string('numeric'),
-        '{}-{}'.format(gen_string('alpha', 5), gen_string('alpha', 5)),
-        '{}_{}'.format(gen_string('alpha', 5), gen_string('alpha', 5)),
+        f"{gen_string('alpha', 5)}-{gen_string('alpha', 5)}",
+        f"{gen_string('alpha', 5)}-{gen_string('alpha', 5)}",
     ]
 
 
-class OrganizationTestCase(CLITestCase):
-    """Tests for Organizations via Hammer CLI"""
+@pytest.fixture
+def _make_proxy(request):
```

Comment:
I'd remove the leading `_`, as fixtures aren't imported between modules anyway.

Reply:
future TODO - use the core pytest fixtures for proxy and org creation, instead of CLI factory methods.

Reply:
he has to keep the `_` because `make_proxy` is already int he namespace because of an import

Reply:
Can just use a different name for the fixture.

Reply:
Is there a problem with the name I chose?

Reply:
I guess you could call it a personal problem. Its an odd pattern for a fixture name, as `_method` implies a 'private' method, which is implicit to defining a pytest fixture within a test module.

Reply:
Changed `_make_org` and `_make_proxy` to `org` and `proxy`.